applications/solvers/additiveFoam/movingHeatSource/heatSourceModels/modifiedSuperGaussian/modifiedSuperGaussian.H

```diff
@@ -93,8 +93,9 @@ public:
 
     // Member Functions
 
-        //- Return the source volumetric heating
-        virtual tmp<volScalarField> qDot() const;
+        inline virtual scalar weight(const vector& d);
```

Comment:
did we ever figure out if these are actually getting inlined?

Reply:
Yeah, they do. You can also see that the movingBeam helper functions are inline now as well